massa-wallet/src/lib.rs

```diff
@@ -19,17 +19,20 @@ use std::collections::hash_map::Entry;
 use std::collections::HashSet;
 use std::path::PathBuf;
 use std::str::FromStr;
+use zeroize::{Zeroize, ZeroizeOnDrop};
 
 mod error;
 
 const WALLET_VERSION: u64 = 1;
 
 /// Contains the keypairs created in the wallet.
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, Serialize, Zeroize, ZeroizeOnDrop)]
 pub struct Wallet {
     /// Keypairs and addresses
+    #[zeroize(skip)]
```

Comment:
I don't get why we would skip fields?
On drop, we want them to be zeroized, don't we?

Reply:
These field types are not supported by Zeroize and what we really want to set to 0 is the password. Note that the KeyPair struct has already Zeroize & ZeroizeDrop features (because the feature "zeroize" is enabled for the crate ed25519)

Reply:
thanks for the explanation